### PR TITLE
Deprecate project entirely.

### LINKED ISSUE #2: Deprecate package
`os.initgroups` was added to the standard library in Python 2.7, which makes this standalone version unneccessary.  For backward compatibility:

- [ ] Drop any support for Python < 2.7.
- [ ] Remove the C code
- [ ] Replace it with wrapped calls to `os.initgroups`.
- [ ] Mark those calls using `zope.deprecation`.
- [ ] Mark the package w/ the "inactive" Trove classifier.

Supersedes #1.

### PR DESCRIPTION
**Pull request by tseaver**
`os.initgroups` was added to the stdlib in Python 2.7.

Closes #2.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,26 +12,35 @@
...
+with open('README.rst') as f:
+    README = f.read()
+
+with open('CHANGES.rst') as f:
+    CHANGES = f.read()
 
 setup(name='initgroups',
-      version = '2.14.0.dev0',
+      version = '3.0.dev0',
       url='http://pypi.python.org/pypi/initgroups',
```
Comment (mgedmin) [inline setup.py]: Perhaps change to `https://github.com/zopefoundation/initgroups`?

File: src/initgroups/__init__.py [hunk 2]
```diff
@@ -12,4 +12,16 @@
...
+import os
+import warnings
+
+
+_MESSAGE =(
+    "The standalone 'initgroups.initgroups' functinon is deprecated.  "
+    "Use 'os.initgroups' instead.")
+
+# We do not use DeprecationWarning because it is ignored by default.
+warnings.warn(_MESSAGE)
```
Comment (mgedmin) [inline src/initgroups/__init__.py]: I don't like this.

I understand why DeprecationWarnings were hidden by default (end-users can't do anything about them), and I don't think removing the use of a thin backwards-compatibility wrapper warrants overriding warning policy that way.

**Review by mgedmin — APPROVED** [review]
lgtm, although I would prefer to see DeprecationWarning used for deprecations.

File: src/initgroups/__init__.py (hunk 2 above)
Comment (tseaver) [reply src/initgroups/__init__.py]: OK, I'll acquiesce.

**Pull request merged**